Dockerfile

```
@@ -1,7 +1,7 @@
 #
 # Builder Image
 #
-FROM gradle:6.5-jdk11 AS builder
+FROM gradle:7.6.1-jdk11 AS builder
```

Does
```suggestion
FROM gradle:7.6.1-jdk17 AS builder
```
exist?  We're building with java 17 these days.

---

Yes, it does
I just tested it with Java 17 and it worked
Thanks for pointing that out!